content/applications/services/helpdesk/overview/ratings.rst

```diff
@@ -1,51 +1,107 @@
-=======
-Ratings
-=======
+================
+Customer ratings
+================
 
-Allow customers to rate their experience with your helpdesk teams to strengthen your credibility and
-gain their trust. Reviews can also influence a customer’s decision and open space for feedback that
-can help you improve the quality of your services.
+Asking customers to rate their experience with :guilabel:`Helpdesk` support provides an opportunity
+to gauge team performance, and track customer satisfaction. Ratings can published to the portal,
+providing customers with a general overview of the team's performance.
 
-Set up
-======
+Enable customer ratings on a Helpdesk team
+==========================================
 
-Go to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams` and enable *Ratings on
-tickets*. The feature automatically adds a default email template on the non-folded *closing
-stage(s)* of that team.
+To enable :guilabel:`Customer Ratings` on a Helpdesk team, go to :menuselection:`Helpdesk -->
+Configuration --> Teams`. Select a team from the list and navigate to the settings page. Scroll to
+the :guilabel:`Performance` section, and check the box for :guilabel:`Customer Ratings`.
 
-.. image:: ratings/ratings_on_tickets.png
+.. image:: ratings/ratings-enable.png
    :align: center
    :alt: Overview of the settings page of a helpdesk team emphasizing the rating on ticket feature
          in Odoo Helpdesk
 
-To edit the email template and the stage(s) set as the closing ones, go to the Kanban view of your
-helpdesk team and click on *Settings*, then on *Edit Stage*.
+Set a ratings request email template on a stage
+===============================================
 
-.. image:: ratings/edit_stage.png
-   :align: center
-   :alt: Overview of a helpdesk team kanban view emphasizing the menu edit stage in Odoo Helpdesk
+To automatically request ratings from customers once their tickets have closed, an email template
+needs to be added to the appropriate stage.
+
+To choose a stage, go to :menuselection:`Configuration --> Stages`. Or, from a team's settings
```

Comment:
Thank you for catching this. I am removing the menu "configuration -> stages" reference, just to avoid any unnecessary confusion/updates. It actually makes more sense to direct users to the stage list from the settings page, because they will theoretically be there anyway if there are following the steps.